Softeq.XToolkit.WhiteLabel/Model/Platform.cs

```diff
@@ -14,15 +15,23 @@ public enum Platform
         Unknown,
         Android,
         iOS,
+
+        [Obsolete]
         WindowsPhone,
         Windows,
+
+        [Obsolete]
```

Comment:
may be make sense remove old types at all?

Reply:
let's make it obsolete for now, and remove it later to be a more smooth migration